io_scene_godot/export_godot.py

```diff
@@ -129,6 +133,14 @@ def export_scene(self):
             if obj in self.valid_nodes and obj.parent is None:
                 self.export_node(obj, root_gd_node)
 
+        # second pass to export animation
```

Comment:
I'm very keen to do the exports in a single "pass" because otherwise you may have issues with (for example) an object's animations being exported without the object itself. 
I think I'm missing the reason why this is necessary, so next time you're on IRC, I think we should discuss this.